week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,25 +24,40 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
+  let newPrice= Math.round(calculateSalesTax(productPrice)*100)/100;
```

Comment:
Can you explain what the `Math.round(...*100)/100` and the `.toFixed(2)` here do?